polynote-runtime/src/main/scala/polynote/runtime/DataEncoder.scala

```diff
@@ -50,14 +51,43 @@ trait DataEncoder[@specialized T] extends Serializable {
 }
 
 object DataEncoder extends DataEncoder0 {
+scala.collection.immutable.HashMap().toString()
+  def truncateString(str: String): String = if (str.length > 255) str.substring(0, 254) + "…" else str
+  def formatCollection[T](coll: GenTraversable[T], formatItem: T => String, prefix: Option[String]): String = {
+    val innerStrs = if (coll.size > 10) {
```

Comment:
Perhaps `10` should be configurable?

Reply:
This is *just* for the `String` representation, so I think it's OK to hard-code it. If you have a collection, we should give you a better way to look at it than the `String` anyhow (if there are cases where we don't, I'd argue _those_ are the problem)